start.sh

```
@@ -9,20 +9,70 @@ if ! command -v go &> /dev/null; then
+    echo "⚡ Execute endpoint: http://localhost:8080/api/execute"
+    echo ""
+    echo "Press Ctrl+C to stop the server"
+    echo "================================"
+
+    # Start server
+    ./main
+else
+    echo "⚠️  Make not found, using fallback build method..."
+
+    # Navigate to API directory
+    cd api
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)